sanitizers/app/build.gradle

```diff
@@ -0,0 +1,76 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+}
+
+android {
+    namespace 'com.example.sanitizers'
+    ndkVersion '25.1.8937393'
+    compileSdk 32
+
+    defaultConfig {
+        applicationId "com.example.sanitizers"
+        minSdk 21
+        targetSdk 32
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+        hwasan {
```

Comment:
Is it possible to use that build.gradle defined virtual device thing to automatically run this on a compatible system image?

Reply:
Maybe. x86_64 and aarch64 both support hwasan. For now, I think I'm just going to point to the hwasan docs in developer.android.com.